package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "rails-template",
+  "version": "1.0.0",
+  "repository": "git@github.com:ackama/rails-template.git",
+  "private": true,
+  "prettier": "prettier-config-ackama",
+  "devDependencies": {
+    "eslint": "^6.7.2",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-eslint-comments": "^3.1.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.17.0",
+    "eslint-plugin-react-hooks": "^2.3.0",
+    "prettier": "^1.19.1",
+    "prettier-config-ackama": "^0.1.0"
+  },
+  "scripts": {
+    "js-lint": "eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts,tsx,jsx",
```

Comment:
Typically these are called `lint` & `format`.

I'm happy w/ `lint`, `lint-fx`, `format`, & `format-fix`.

I don't think we should be worried about these clashing w/ linting of Ruby, since `package.json` is a JS concept.